src/main.js

```diff
@@ -17,39 +21,93 @@ const filmCards = new Array(FILM_COUNT)
   .map((element, idElement) => generateCardFilm(idElement));
 const mainElement = document.querySelector('.main');
 const headerElement = document.querySelector('.header');
+const page = document.querySelector('body');
 
-render(headerElement, createUserProfileTemplate, 'beforeend');
-render(mainElement, createMenuTemplate(filmCards), 'beforeend');
-render(mainElement, createSortCardTemplate, 'beforeend');
-render(mainElement, '<section class="films"></section>', 'beforeend');
+renderElement(headerElement, new UserProfileView().getElement(), RenderPosition.BEFOREEND);
```

Comment:
```suggestion
const userProfileView = new UserProfileView();
renderElement(headerElement, userProfileView.getElement(), RenderPosition.BEFOREEND);
```

ну и со всеми остальными аналогично бы переделать